source/NationalInstruments/jquery.flot.scattergraph.js

```diff
@@ -516,6 +517,40 @@ THE SOFTWARE.
 
                 ctx.restore();
             };
+
+            function findNearbyItems (plot, canvasX, canvasY, series, seriesIndex, radius, computeDistance, items) {
+                var ps = series[seriesIndex].datapoints.pointsize;
```

Comment:
@atmgrifter00 I thought the hook was supposed to only pass in a single series? That seems to be what the API documentation implies.

Reply:
Yeah, this was a goof in the implementation, which sucks because this is a breaking API change if we decide to change it. So, I'd like @andrewdove1's  perspective on how to move forward here. We can keep the API as-is, which is functional, but probably not ideal, and then change the docs. Or, we change the API, and then make a call on how we want to version flot (major or patch).

Reply:
Let's leave it and change the docs. Seems like having access to all the series here might be useful at some point (no pun intended)